src/seedu/addressbook/commands/updateAttendanceCommand.java

```diff
@@ -0,0 +1,46 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Attendance;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ *  Clocks in the date and time where the student is present.
+ */
+public class updateAttendanceCommand extends Command {
+    public static final String COMMAND_WORD = "attendance";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n" + "Updates the attendance of a student. \n"
+            + "Parameters: n/STUDENTNAME att/ATTENDANCE \n\t"
+            + "Example: " + COMMAND_WORD + " "
+            + "n/Neow Bo Kai att/1 or 0"; // '1' will be read at true, '0' will be read as false
+
+    public static final String MESSAGE_SUCCESS = "Attendance updated : %1$s";
+    public static final String MESSAGE_DUPLICATE_EXAM = "Attendance has already been taken";
+
+    private static String currentDate = new SimpleDateFormat("dd-MM-yyyy").format(new Date());
+
+    public String isPresent;
+    public String indexOfPerson;
+
+    // Constructor
+    public updateAttendanceCommand(String indexOfPerson, String isPresent) {
+        this.isPresent = isPresent;
+        this.indexOfPerson = indexOfPerson;
+    }
+
+    @Override
+    public CommandResult execute() {
+            Attendance.addAttendance(currentDate, isPresent);
```

Comment:
final ReadOnlyPerson target = getTargetPerson(); //first you find the person from the list
Person toEdit = addressBook.findPerson(target); //find and extract out the Person in addressBook
toEdit.updateAttendance(currentDate, isPresent); //calling the method to change attendance in Person toEdit